js/admin.js

```diff
@@ -1,4 +1,13 @@
 jQuery(document).ready(function($){
+	if ( sessionStorage.getItem('message_save_view') ) {
```

Comment:
I think it would be better to implement this on the server-side instead of in JS. There are some edge cases, like if I reload the page and resend the form, where this value will not be set on the client-side but the server will actually resave the form.

Could you implement this on the PHP-side that when the form is saved this notice is displayed?

Reply:
Maybe I misunderstand this comment. As I know we always save the message to the sessionStorage after submitting the form. The case you mention above can't happen. For the PHP-side solution, what is the steps to be done? After submitting the form, the page will be redirected to edit page. Maybe we need the query params to check if you decide to work on PHP-side.

Reply:
@jmaas-onoffice did you take a look this one?

Reply:
So right now this is the mechanism:

1. Press "Save".
2. Store that we just clicked "Save" in sessionStorage.
3. Send the form to WordPress.
4. WordPress returns the new edit page.
5. We check sessionStorage to see if we need to display the message.

But why do we need to do this in JavaScript? Can't we check in PHP whether we just saved and display the message directly in PHP, instead of using JavaScript?

So the mechanism would be:

1. Press "Save".
2. The form is sent to WordPress.
3. In PHP when we are saving the form, we also display the "List saved" message.

Here, no JavaScript is needed and we don't need to track button clicks, because in PHP we know whether we are saving the form or not.

Is this a bit clearer?